packages/react-app/src/components/HardhatEVM.jsx

```diff
@@ -0,0 +1,175 @@
+import { SendOutlined } from "@ant-design/icons";
+import { Button, Input, Tooltip, Drawer, Form, Row, Col, Select, DatePicker, Divider } from "antd";
+import { useLookupAddress } from "eth-hooks";
+import React, { useCallback, useEffect, useState } from "react";
+import Blockies from "react-blockies";
+import { Transactor } from "../helpers";
+import Wallet from "./Wallet";
+import {
+  useOnBlock,
+} from "../hooks";
+import { formatEther, parseEther } from "@ethersproject/units";
+import { parse } from "graphql";
+
+const { Option } = Select;
+const { utils, BigNumber } = require("ethers");
+
+/*
+  ~ What it does? ~
+
+  
+
+  ~ How can I use? ~
+
+  <HardhatEVM
+    localProvider={localProvider}
+  />
+
+  ~ Features ~
+*/
+
+export default function HardhatEVM(props) {
+  const [address, setAddress] = useState();
+  const [visible, setVisible] = useState(false);
+  const [chainTimestamp, setChainTimestamp] = useState();
+  const [accountBalance, setAccountBalance] = useState();
+
+  const localProvider = props.localProvider;
```

Comment:
there doesn't seem to be a reason to only destructure one of the prop vars

Reply:
What would be the right way to do it? :)

Reply:
Oh what I meant was we should destructure al the variables or none of them and use props directly. It’s inconsistent to do half way

Reply:
ah, gotcha, thanks!